server/code/schema/README.txt

```diff
@@ -1,3 +0,0 @@
-This directory contains third party schema.
```

Comment:
Looks like the whole 'schema' directory got removed (as opposed to just the oracle script). I'm unable to see how/if they're currently used. Do we not need these scripts any more?

Reply:
Yes, they are unused.  Everything is in liquibase now.